dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/models/SubSnpCoreFieldsTest.java

```diff
@@ -130,72 +130,97 @@ public void failWithNegativeChromosomeCoordinates() {
     }
 
     @Test
-    public void testGenotypeList() {
-
+    public void testGenotypes_Allele_Reverse() {
```

Comment:
Given that other methods below include `Multiallelic` in the name, I think what you meant here was `Biallellic` instead of `Allele`.